src/typing_extensions/ext.py

```diff
@@ -0,0 +1,99 @@
+"""
+Protocols and type aliases that are not scheduled for inclusion in typing.
+"""
+
+from os import PathLike
+from typing import AbstractSet, Awaitable, Container, Iterable, Tuple, TypeVar, Union
+from typing_extensions import Protocol, TypeAlias
+
+_T_co = TypeVar("_T_co", covariant=True)
+_T_contra = TypeVar("_T_contra", contravariant=True)
+_KT = TypeVar("_KT")
+_KT_co = TypeVar("_KT_co", covariant=True)
+_KT_contra = TypeVar("_KT_contra", contravariant=True)
+_VT_co = TypeVar("_VT_co", covariant=True)
+
+#
+# Protocols for dunder methods
+#
+
+
+class SupportsAnext(Protocol[_T_co]):
+    def __anext__(self) -> Awaitable[_T_co]:
+        ...
+
+
+class SupportsDivMod(Protocol[_T_contra, _T_co]):
+    def __divmod__(self, __other: _T_contra) -> _T_co:
+        ...
+
+
+class SupportsGetItem(Container[_KT_contra], Protocol[_KT_contra, _VT_co]):
```

Comment:
I don't know if inheriting from Container is right. A type that supports `__getitem__` also automatically supports `in`, but type checkers may not know that, so a type providing only `__getitem__` may get rejected from this protocol.